sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/autorest.md

```diff
@@ -147,6 +147,11 @@ rename-mapping:
   CassandraTablePropertiesResource: ExtendedCassandraTableResourceInfo
   CassandraTableResource: CassandraTableResourceInfo
   CosmosTablePropertiesResource: ExtendedCosmosTableResourceInfo
+  ClientEncryptionKeyGetPropertiesResource: CosmosDBSqlClientEncryptionKeyProperties
+  ClientEncryptionKeyResource: CosmosDBSqlClientEncryptionKeyResource
```

Comment:
As I claimed, `Resource` is a reserved suffix in track 2 SDK, please rename this to `CosmosDBSqlClientEncryptionKeyProperties` or `CosmosDBSqlClientEncryptionKeyResourceInfo` to align with other classes in this SDK

Reply:
@ArcturusZhang   fixed